package/src/languageService/kustoLanguageService.ts

```diff
@@ -1282,13 +1282,26 @@ class KustoLanguageService implements LanguageService {
 
         const quickInfo = currentBLock.Service.GetQuickInfo(cursorOffset);
 
-        if (!quickInfo || !quickInfo.Text) {
+        if (!quickInfo || !quickInfo.Items) {
             return Promise.resolve(undefined);
         }
 
+        let items = this.toArray(quickInfo.Items)
+
+        if (!items) {
+            return Promise.resolve(undefined);
```

Comment:
There's a lot of `Promise.resolve` in this file. Maybe make the method async?

Reply:
for historical reasons there is no async await in this codebase at all (i do not believe we're polyfilling it for older browsers).  but it's a good suggestion. 
At this point since we have people waiting on this fix, I don't want to make this call but will consider for future PRs